example-junit/src/test/java/com/epam/reportportal/example/junit/extension/JUnitRetryImpl.java

```diff
@@ -3,10 +3,10 @@
 import com.nordstrom.automation.junit.JUnitRetryAnalyzer;
 import org.junit.runners.model.FrameworkMethod;
 
-public class RetryImpl implements JUnitRetryAnalyzer {
+public class JUnitRetryImpl implements JUnitRetryAnalyzer {
 
 	@Override
 	public boolean retry(FrameworkMethod method, Throwable thrown) {
-		return thrown != null;
+		return true;
```

Comment:
With this hardcoded result, failed tests will always be retried. In this instance, we could also limit the retriable tests to the `RetryTest` class:
```java
@Override
public boolean retry(FrameworkMethod method, Throwable thrown) {
    return method.getDeclaringClass() == RetryTest.class;
}
```